src/transformers/models/distilbert/modeling_distilbert.py

```diff
@@ -145,7 +154,19 @@ def prune_heads(self, heads):
         self.dim = attention_head_size * self.n_heads
         self.pruned_heads = self.pruned_heads.union(heads)
 
-    def forward(self, query, key, value, mask, head_mask=None, output_attentions=False):
+    def forward(
+        self,
+        query,
+        key,
+        value,
+        mask,
+        attention_mask=None,
```

Comment:
mask and `attention_mask` seem to be doing the same thing, just renaming mask to `attention_mask` would be sufficient.